src/zzz_od/api/routers/battle_assistant.py

```
@@ -0,0 +1,2472 @@
+            # 验证手柄类型是否受支持
+            supported_gamepad_types = ["none", "xbox", "ds4"]
+            if settings.gamepad_type not in supported_gamepad_types:
+                raise HTTPException(
+                    status_code=400,
+                    detail={
+                        "error": {
+                            "code": "UNSUPPORTED_GAMEPAD_TYPE",
+                            "message": f"不支持的手柄类型: {settings.gamepad_type}"
+                        }
+                    }
+                )
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)